design/ceph/multus-network.md

```diff
@@ -78,7 +78,22 @@ Nothing to do in particular since they don't use any service IPs.
 
 ### CSI pods
 
-We can add annotations to these pods and they can reach out the Ceph public network, then the driver will expose the block or the filesystem normally.
+The CSI pods will run in the node's host network namespace.
+
+When deploying a CephCluster resource configured to use multus networks, a multus-connected network interface will be added to the host network namespace of all nodes that will run CSI plugin pods.
```

Comment:
After the cluster is started, what happens when a new node is added to the cluster? Will it be a problem that the daemonset for the csi volume plugins will start before the job is triggered to configure the multus interface?

Reply:
The csi plugin won't work properly if the job that multus interface isn't run first. The code doesn't handle the addition of new nodes, I need to add this.